BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/Dtos/ParticipantDto.cs

```diff
@@ -27,6 +27,8 @@ public enum CaseRoleGroup
         Respondent = 4,
         Observer = 5,
         PanelMember = 6,
-        None = 7
+        Appellant = 7,
```

Comment:
Let's keep none as 7 and add the two new ones at the End. Just in case there are any existing data.